test/integration/metrics-compliance-static-analysis.py

```
@@ -0,0 +1,375 @@
+        self.servicemonitor_file = (
+            self.base_path / "manifests/base/monitoring/servicemonitor.yaml"
+        )
+        self.recording_rules_file = (
+            self.base_path / "manifests/base/monitoring/recording-rules.yaml"
+        )
+        self.test_results = []
+        
+    def run_command(self, cmd, timeout=5):
+        """Execute shell command and return output"""
+        try:
+            result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)